pkg/authorization/api/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package api is the internal version of the API.
+// +genconversion=true
```

Comment:
actually I can use this beautiful system of comments and set it to not generate conversions for api.builds (the generated conversion is empty)